app/src/main/java/com/hedvig/app/feature/offer/OfferRepository.kt

```diff
@@ -43,13 +44,7 @@ class OfferRepository(
         response.errors?.let {
             return OfferResult.Error(it.firstOrNull()?.message)
         }
-
         val data = response.data ?: return OfferResult.Error()
-
-        if (data.contracts.isNotEmpty()) {
-            return OfferResult.HasContracts
```

Comment:
I ran into problems with this error - in moving flow we will have contracts but still want to fetch an offer. When do we need to handle this case? If the user tries to onboard a second time?

Reply:
How would you get stuck post-sign? Because of bank-id callbacks and stuff?